_posts/zh/newsletters/2024-01-17-newsletter.md

```diff
@@ -0,0 +1,111 @@
+
+- [Bitcoin Core #29200][] 允许 [I2P 网络支持][topic anonymity networks]使用 “ECIES-X25519 和 ElGamal（分别为 4 型和 0 型）加密连接。这允许连接到任一类型的 I2P 对等节点，首选其中更新、更快的 ECIES-X25519”。
+
+- [Bitcoin Core #28890][] 删除了之前被废弃的 `-rpcserialversion` 配置参数（参见[周报 #269][news269 rpc]）。该选项是在向 v0 segwit 过渡期间引入的，目的是允许旧版程序继续以剥离格式（不含任何 segwit 字段）访问区块和交易。目前，所有程序都应更新以处理 segwit 交易，不再需要该选项。
+
+- [Eclair #2808][] 更新了 `open` 命令，增加了一个 `--fundingFeeBudgetSatoshis` 参数，用于定义节点为打开通道而愿意支付的链上费用上限，默认设置为向通道支付的金额的 0.1% 。如果可能，Eclair 会尽量支付较低的费用，但必要时也会支付预算内的最高金额。`rbfopen` 命令也进行了更新，以接受相同的参数，该参数定义了 [RBF 费用提升][topic rbf]的最大金额。
+
+- [LND #8188][] 添加了几个新的 RPC，用于快速获取调试信息，用公钥为其加密，并根据私钥进行解密。正如 PR 中解释的那样：“我们的想法是在 GitHub issue 模板中发布一个公钥，并要求用户运行 `lncli encryptdebugpackage` 命令，然后将加密的输出文件上传到 GitHub issue，为我们提供调试用户问题时通常需要的信息。”
+
+- [LND #8096][] 添加了 “手续费峰值缓冲”。在当前的闪电网络协议中，通道注资方负责支付直接包含在承诺交易和预签的 HTLC-Success 和 HTLC-Timeout 交易（HTLC-X 交易）中的任何链上手续费。如果通道注资方资金不足并且费率上升，注资方可能会因为没有足够的资金支付手续费而无法接受新的付款——尽管如果付款结算，新的付款会增加注资方的余额。为避免此类通道注资问题，[BOLT2][]（几年前在 [BOLTs #740][] 中添加）中的一项建议提出，注资方应自愿保留额外的资金储备，以确保即使在费用增加的情况下也能收到额外的付款。LND 现在实施了这一解决方案，Core Lightning 和 Eclair 也实施了这一解决方案（见第 [85][news85 stuck] 期和第 [89][news89 stuck] 期周报）。
+
+- [LND #8095][] 和 [#8142][lnd #8142] 代码库的部分内容添加了处理[盲化路径][topic rv routing]的附加逻辑。这是正在进行中的让 LND 全面支持盲化路径工作的一部分。
+
+{% assign day_after_posting = page.date | date: "%s" | plus: 86400 | date: "%Y-%m-%d 15:00" %}
+{% include snippets/recap-ad.md when=day_after_posting %}
+{% include references.md %}
+{% include linkers/issues.md v=2 issues="28134,29058,29200,28890,2808,8188,8096,8095,8142,740" %}
+[morehouse delving]: https://delvingbitcoin.org/t/dos-disclosure-channel-open-race-in-cln/385
+[morehouse blog]: https://morehouse.github.io/lightning/cln-channel-open-race/
+[news266 dos]: /zh/newsletters/2023/08/30/#disclosure-of-past-ln-vulnerability-related-to-fake-funding
+[script wiki]: https://en.bitcoin.it/wiki/Script#Arithmetic
+[news166 tluv]: /en/newsletters/2021/09/15/#amount-introspection
+[news283 exits]: /zh/newsletters/2024/01/03/#pool-exit-payment-batching-with-delegation-using-fraud-proofs
+[daftuar cluster]: https://delvingbitcoin.org/t/an-overview-of-the-cluster-mempool-proposal/393/
+[news280 cluster]: /zh/newsletters/2023/12/06/#cluster-mempool-discussion
+[news267 compress]: /zh/newsletters/2023/09/06/#bitcoin-transaction-compression
+[briar compress]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2024-January/022274.html
+[compress spec]: https://github.com/bitcoin/bitcoin/blob/7e8511c1a8229736d58bd904595815636f410aa8/doc/compressed_transactions.md
+[news201 mev]: /en/newsletters/2022/05/25/#miner-extractable-value-discussion
+[news266 lnbugs]: /zh/newsletters/2023/08/30/#disclosure-of-past-ln-vulnerability-related-to-fake-funding
+[race condition]: https://en.wikipedia.org/wiki/Race_condition
```

Comment:
@hulatown Does this work better: https://zh.wikipedia.org/wiki/%E7%AB%B6%E7%88%AD%E5%8D%B1%E5%AE%B3

Reply:
Yes, this one works!

> @hulatown Does this work better: https://zh.wikipedia.org/wiki/%E7%AB%B6%E7%88%AD%E5%8D%B1%E5%AE%B3